Sources/Components/ARCListItem.swift

```diff
@@ -12,10 +12,16 @@ import CubeFoundationSwiftUI
 /// List Item Component
 public struct ARCListItem<Leading: View, Trailing: View>: View {
 
+    /// Functional
     public var title: String
     public var subtitle: String?
     public var labelTitle: String?
     public var onTap: () -> Void
+    /// Styling
+    public var background: Color?
+    public var titleForeground: Color?
+    public var subtitleForeground: Color?
+    public var verticalPadding: CGFloat?
```

Comment:
Is this one design dept? *Should* we allow this to change?

Reply:
![Screenshot 2023-06-19 at 13 24 15](https://github.com/3sidedcube/ArcUI/assets/33767184/9f7f6166-ce5e-4ab1-8404-ef4082595d69)

@BenShutt Think it's more design vision rather than debt. Bet they wanted to have main principal buttons and secondary smaller one.